models/cont_delay_connection.h

```diff
@@ -141,6 +141,12 @@ class ContDelayConnection : public Connection< targetidentifierT >
   void set_status( const DictionaryDatum& d, ConnectorModel& cm );
 
   /**
+   * Check syn_spec dictionary for parameters that are not allowed for this
+   * connection. Will issue warning or throw error if a parameter is found.
```

Comment:
- this general explanation should be in the base class only (`Connection`)
- here, you could just mention what is illegal for this particular synapse model
- Doxygen comments should begin with a single-line summary followed by details
- "an illegal parameter is found"
- add info that the method does nothing otherwise